lms/static/js/ccx/schedule.js

```diff
@@ -331,6 +356,10 @@ var edx = edx || {};
 	      });
 	},
 
+	disabled_fields: function($selector, disabled) {
```

Comment:
Also, the Javascript style is camelCase, so the functions should be named `disableFields()` and `enableFields()`. Yes, it's different from Python.